tests/kola/data/commonlib.sh

```diff
@@ -80,6 +80,13 @@ is_rhcos9() {
     { [ "${ID}" == "rhel" ] && [ "${VERSION_ID%%.*}" -eq 9 ]; }
 }
 
+# rhcos10 && SCOS 10
+is_el10() {
+    source /etc/os-release
+    { [ "${ID}" == "rhcos" ] && [ "${RHEL_VERSION%%.*}" -eq 10 ]; } || \
```

Comment:
This needs to be

```suggestion
    { [ "${ID}" == "rhel" ] && [ "${VERSION_ID%%.*}" -eq 10 ]; } || \
```

We don't modify `/etc/os-release` anymore in the new model. (See https://github.com/openshift/enhancements/blob/master/enhancements/rhcos/split-rhcos-into-layers.md#etcos-release).

Reply:
ah yes, thanks